### DISCUSSION TITLE
ScriptingJitCompiler - Fails if expression has more than 17 parameters

### DISCUSSION THREAD

**Discussion by mishrashash** [UiPath/corewf#156] [General]
First of all many thanks for the great library!

For a complex expression which has more than 17 parameters e.g. concatenating 18 different string arguments together cause 
CreateExpressionCode to generate invalid Func delegate causing ScriptingAotCompiler BuildAssembly function to fail.

**Comment by lbargaoanu** [comment]
By design :) This is an assumed limitation. But if you have ideas, we're listening.

**Reply by mishrashash** [reply]
Thanks Lucian for the quick response :) Sure I'll open a PR once I get around this.
